WebApp/autoreduce_webapp/static/css/help.css

```diff
@@ -3,7 +3,7 @@
 }
 
 /* Used for a top margin when using anchor links */
-h4::before {
+h3::before {
     display: block;
     content: " ";
     margin-top: -32px;
```

Comment:
This is very minor and a personal preference but since `<h3>` is bigger than `<h4>` the margin also needs to be increased from `32px` to something like `40px`.

32px:
![image](https://user-images.githubusercontent.com/5902731/95593959-39df8000-0a42-11eb-831f-a489ef8396ef.png)

40px:
![image](https://user-images.githubusercontent.com/5902731/95594057-5bd90280-0a42-11eb-99ec-ece900f5c60a.png)

Reply:
This is why I hate doing front end 😛

Reply:
100% agree with you there!